docs/source-app/glossary/storage/drive_content.rst

```diff
@@ -21,15 +32,28 @@ What Drive does for you
 Think of every instance of the Drive object acting like a Google Drive or like Dropbox.
 
 By sharing the Drive between components through the LightningFlow,
-several components can have a shared place to read and write files from.
+several components can have a shared place to read (``s3://`` Drives) or read and write (``lit://`` Drives) files from.
+
+Limitations
+^^^^^^^^^^^
+
+These limitations only apply to ``s3://`` Drives:
+
+* There is no top level “shareable” S3 drive object. Each S3 Drive is owned by a particular Work.
+
+* S3 buckets cannot be mounted as Drives at the runtime of a Work. The `Drive` object must be initialized in a Flow and passed to a Work through its initialization arguments.
+
+* Whenever a Drive is mounted to a Work, the indexing process will be done again for the provided S3 bucket. This may lead to performance issues with particularly large S3 buckets. For context, 1M files with 2-3 levels of nesting takes less than 1 second to index.
```

Comment:
“the indexing process” -> “an indexing process” if we didn’t talk about it earlier

Reply:
Updated